Core/Core/Features/Program/Course/GetHProgramCourseUseCase.swift

```diff
@@ -93,7 +93,7 @@ public final class GetHProgramCourseUseCase: APIUseCase {
                         .setFailureType(to: Error.self)
                         .eraseToAnyPublisher()
                 }
-                return self.fetchCombine(request: GetCoursesEnrollmentIDRequest(userId: userId))
+                return self.fetchCombine(request: HGetCoursesEnrollmentIDRequest(userId: userId))
```

Comment:
The pattern here is `GetHCoursesEnrollmentIDRequest` just like you did with `GetHProgramCourseUseCase`.